jpo-geojsonconverter/src/main/resources/schemas/processed-map.schema.json

```diff
@@ -185,7 +186,7 @@
                     "description": "DE_MsgCount for intersection"
                 },
                 "refPoint": {
-                    "$ref": "https://raw.githubusercontent.com/usdot-jpo-ode/jpo-geojsonconverter/develop/jpo-geojsonconverter/src/main/resources/schemas/map.schema.json#/$defs/OdePosition3D"
+                    "$ref": "#/$defs/Position3D"
```

Comment:
Why copy the definition of Position 3D instead of referencing as before?

Reply:
Same here.